crates/commitlog/src/dio/mod.rs

```diff
@@ -0,0 +1,111 @@
+mod page;
```

Comment:
Yes, I shall add some prose here explaining what direct I/O is and what's the deal with the alignment requirements.